<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPS</class>
 <widget class="QMainWindow" name="XPS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XPS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>工具栏</string>
    </property>
    <widget class="QPushButton" name="pen_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back/Agg2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="erase_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/back/16pic_3839444_b.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="font_sli">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="color_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
</string>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>591</width>
      <height>531</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="Ximage" name="image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>589</width>
       <height>529</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_act"/>
    <addaction name="save_act"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_act">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save_act">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ximage</class>
   <extends>QWidget</extends>
   <header location="global">ximage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>setPen()</slot>
 </slots>
</ui>
